src/components/SendByScan.js

```diff
@@ -115,8 +115,8 @@ class SendByScan extends Component {
 
         if (dataSplit.length === 1 && Web3.utils.isAddress(dataSplit[0])) {
           console.log("RETURN STATE:",this.props.returnState)
-          let returnState = this.props.parseAndCleanPath(dataAfterColon)
-          this.props.returnToState(returnState)
+          let returnState = this.props.parseAndCleanMyAss(dataAfterColon)
+          this.props.returnToState(returnState, this.props.returnState.goBackView)
```

Comment:
I did `git add -p` and pressed s to split the 💩 (chunk), but git told me

```
Stage this 💩  [y,n,q,a,d,e,?]? s
Sorry, cannot split this hunk
```

😭